static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -261,6 +261,7 @@ export function getCompoundDiseaseTreatment(
       clinicalPhaseNumber: Number(element.clinicalPhaseNumber),
     }));
   processedData.sort((a, b) => b.clinicalPhaseNumber - a.clinicalPhaseNumber);
+  console.log(processedData);
```

Comment:
remove this console log?

Reply:
So sorry about the console statements. Removed em!